src/renderer/experimental.ts

```diff
@@ -0,0 +1,35 @@
+import { callDcMethodAsync } from './ipc'
+import { getLogger } from '../shared/logger'
+
+const log = getLogger('renderer/experiments')
+
+class Experimental {
+  help = `
+These functions are highly experimental, use at your own risk.
+- importContacts (contacts:[email,name][]) // for mass importing contacts
+    example: type "exp.importContacts([['email1@example.com', 'Heinz Herlich'],['bea@example.com','Berta Bissig']])"
```

Comment:
Why is this string in the class and not locales/_experimental?

Reply:
because it describes something experimental and needs to be 100% up to date with the code it is describing.

Reply:
That's fine but still it seems to be weird to me to have a translation in the code itself. I mean we can have them in locales/experimental_en.json and just never put that one into the official language repo?

Reply:
we could also add a markdown file to the docs folder...